Formula/c/conda-lock.rb

```diff
@@ -17,6 +17,7 @@ class CondaLock < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:  "6ebb9cdb5410373413983fd3454078ab36a7cc350b0695594a7d406bfcc15f8d"
   end
 
+  depends_on "python-setuptools" => :build
```

Comment:
might be better to just have the formula managing setuptools and once they dont rely on it , it would no longer be part of the resource list.

Reply:
yeah, we had mixed practices on such before, but in the end, it would be better to vendor out the dependency list (unless it is super expensive to build or has rust build dependency to it)